src/backend/fold.js

```diff
@@ -42,6 +43,54 @@ function checkNullHtml(html) {
   return (html === '');
 }
 
+function getHoursFromTime(time) {
+  return time.split(':')[0];
+}
+
+function getMinutessFromTime(time) {
+  return time.split(':')[1];
+}
+
+function getTimeDifferenceOfDates(startTime, sessionTime) {
+  const firstDate = new Date(new Date().getFullYear(), new Date().getMonth(), new Date().getDate(), parseInt(getHoursFromTime(startTime)), parseInt(getMinutessFromTime(startTime)), 0);
+  const secondDate = new Date(new Date().getFullYear(), new Date().getMonth(), new Date().getDate(), parseInt(getHoursFromTime(sessionTime)), parseInt(getMinutessFromTime(sessionTime)), 0);
+  return (secondDate - firstDate);
+}
+
+function convertTimeToPixel(startTime, sessionTime) {
+  const timeDiff = getTimeDifferenceOfDates(startTime, sessionTime);
+
+  const top = timeDiff * timeToPixel/ (1000 * 60 * 15) + timeToPixel; // distance of session from top of the table
```

Comment:
@geekyd How have you calculated the top? According to my approach =>
1. We should see the box height in pixels, let's say it as 100 px in height.
2. Now each 15minutes ==> 100 px height. 1 min  = 100/15 px, let say it s.
3. Calculating date difference, changing into minutes and multiplying by s.
 @geekyd Is your approach different ?